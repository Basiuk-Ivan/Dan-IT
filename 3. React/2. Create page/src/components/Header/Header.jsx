import { Component } from 'react';
import PropTypes from 'prop-types';
import HeaderMenu from './Header components/HeaderMenu';
import HeaderInfo from './Header components/HeaderInfo';
import HeaderLogo from './Header components/HeaderLogo/HeaderLogo';
import s from '../../style/header.module.scss';

class Header extends Component {
    render() {
        const { favouriteItems, cartItems, parseLocalStorage, parseLocalStorageFavourite } = this.props;
        return (
            <header className={s.header}>
                <HeaderMenu />
                <HeaderLogo />
                <HeaderInfo {...this.props} />
            </header>
        );
    }
}

Header.propTypes = {
    favouriteItems: PropTypes.array,
    cartItems: PropTypes.array,
    parseLocalStorage: PropTypes.func,
    parseLocalStorageFavourite: PropTypes.func,
};

export default Header;
