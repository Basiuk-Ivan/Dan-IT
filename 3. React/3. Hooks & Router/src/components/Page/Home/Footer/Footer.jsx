import footerLogo from '../../../../assets/img/Page/footer/footerLogo.svg';
import s from '../../../../style/components/footer.module.scss';

const Footer = () => {
    return (
        <>
            <div className={s.footer}>
                <div className={s.footerContact}>
                    <div className={s.footerLogo}>
                        <img src={footerLogo} alt={'footerLogo'}></img>
                    </div>
                    <div className={s.footerContacts}>
                        <p>Потрібна допомога з замовленням?</p>
                        <p>
                            <a href="tel:+380993242484">+380 99 324 24 84</a>
                        </p>
                        <p>
                            <a href="mailto:info@loylen.com">info@loylen.com</a>
                        </p>
                    </div>
                    <div className={s.footerDesk}>
                        <p>2021 Інтернет-магазин LOYLEN</p>
                        <p>Всі права захищені</p>
                        <p>Розробка дизайну</p>
                    </div>
                </div>
                <div className={s.footerInfo}>
                    <div className={s.footerInfoWrapp}>
                        <div className={s.footerInfoCatalog}>
                            <div className={s.footerInfoCatalogTitle}>Каталог</div>
                            <ul className={s.footerInfoText}>
                                <li>
                                    <a href="#">Новинки</a>
                                </li>
                                <li>
                                    <a href="#">Розпродаж</a>
                                </li>
                                <li>
                                    <a href="#">Готові зразки</a>
                                </li>
                                <li>
                                    <a href="#">Всі товари</a>
                                </li>
                            </ul>
                        </div>
                        <div className={s.footerInfoCustomer}>
                            <div className={s.footerInfoCatalogTitle}>Покупець</div>
                            <ul className={s.footerInfoText}>
                                <li>
                                    <a href="#">Особистий кабінет</a>
                                </li>
                                <li>
                                    <a href="#">Таблиця розмірів</a>
                                </li>
                                <li>
                                    <a href="#">Доставка і оплата</a>
                                </li>
                                <li>
                                    <a href="#">Повернення</a>
                                </li>
                            </ul>
                        </div>
                        <div className={s.footerInfoAbout}>
                            <div className={s.footerInfoCatalogTitle}>Про нас</div>
                            <ul className={s.footerInfoText}>
                                <li>
                                    <a href="#">Про бренд</a>
                                </li>
                                <li>
                                    <a href="#">Контакти</a>
                                </li>
                                <li>
                                    <a href="#">Оферта</a>
                                </li>
                                <li>
                                    <a href="#">Політика</a>
                                </li>
                            </ul>
                        </div>
                    </div>
                </div>
            </div>
        </>
    );
};

export default Footer;
