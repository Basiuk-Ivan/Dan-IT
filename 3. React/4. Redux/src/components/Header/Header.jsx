import HeaderMenu from './Header components/HeaderMenu';
import HeaderInfo from './Header components/HeaderInfo';
import HeaderLogo from './Header components/HeaderLogo/HeaderLogo';
import s from '../../style/components/header.module.scss';

const Header = () => {
    return (
        <header className={s.header}>
            <HeaderMenu />
            <HeaderLogo />
            <HeaderInfo />
        </header>
    );
};

export default Header;
