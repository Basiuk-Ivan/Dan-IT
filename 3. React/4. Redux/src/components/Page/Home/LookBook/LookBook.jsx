import lookBook from '../../../../assets/img/Page/lookBook/lookBook.png';
import s from '../../../../style/components/pageHome/lookBook.module.scss';

const LookBook = () => {
    return (
        <>
            <div className={s.lookBook}>
                <div className={s.lookBookImg}>
                    <img src={lookBook} alt="lookBook"></img>
                </div>
                <div className={s.lookBookInfo}>
                    <div className={s.lookBookInfoWrapp}>
                        <div className={s.lookBookTitle}>LOOKBOOK</div>
                        <div className={s.lookBookText}>При створені коллекцій ми враховуємо, щоб всі вироби поєднувались. Це дозволяє підібрати повний образ на всі випадки життя.</div>
                        <button className={s.lookBookButton}>Дивитись</button>
                    </div>
                </div>
            </div>
        </>
    );
};

export default LookBook;
