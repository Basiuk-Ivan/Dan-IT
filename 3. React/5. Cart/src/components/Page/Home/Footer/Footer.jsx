import footerLogo from '../../../../assets/img/Page/footer/footerLogo.svg';
import s from '../../../../style/components/footer.module.scss';

const Footer = () => {
    const footerDesk = ['2021 Інтернет-магазин LOYLEN', 'Всі права захищені', 'Розробка дизайну'];
    const footerTextCatalog = ['Новинки', 'Розпродаж', 'Готові зразки', 'Всі товари'];
    const footerTextCustomer = ['Особистий кабінет', 'Таблиця розмірів', 'Доставка і оплата', 'Повернення'];
    const footerTextAboutUs = ['Про бренд', 'Контакти', 'Оферта', 'Політика'];

    return (
        <>
            <div className={s.footer}>
                <div className={s.footerContact}>
                    <div className={s.footerLogo}>
                        <img src={footerLogo} alt={'footerLogo'}></img>
                    </div>
                    <div className={s.footerContacts}>
                        <p>Потрібна допомога з замовленням?</p>
                        <p>
                            <a href="tel:+380993242484">+380 99 324 24 84</a>
                        </p>
                        <p>
                            <a href="mailto:info@loylen.com">info@loylen.com</a>
                        </p>
                    </div>
                    <div className={s.footerDesk}>
                        {footerDesk.map(item => (
                            <p key={item}>{item}</p>
                        ))}
                    </div>
                </div>
                <div className={s.footerInfo}>
                    <div className={s.footerInfoWrapp}>
                        <div className={s.footerInfoCatalog}>
                            <div className={s.footerInfoCatalogTitle}>Каталог</div>
                            <ul className={s.footerInfoText}>
                                {footerTextCatalog.map(item => (
                                    <li key={item}>
                                        <a href="#">{item}</a>
                                    </li>
                                ))}
                            </ul>
                        </div>
                        <div className={s.footerInfoCustomer}>
                            <div className={s.footerInfoCatalogTitle}>Покупець</div>
                            <ul className={s.footerInfoText}>
                                {footerTextCustomer.map(item => (
                                    <li key={item}>
                                        <a href="#">{item}</a>
                                    </li>
                                ))}
                            </ul>
                        </div>
                        <div className={s.footerInfoAbout}>
                            <div className={s.footerInfoCatalogTitle}>Про нас</div>
                            <ul className={s.footerInfoText}>
                                {footerTextAboutUs.map(item => (
                                    <li key={item}>
                                        <a href="#">{item}</a>
                                    </li>
                                ))}
                            </ul>
                        </div>
                    </div>
                </div>
            </div>
        </>
    );
};

export default Footer;
