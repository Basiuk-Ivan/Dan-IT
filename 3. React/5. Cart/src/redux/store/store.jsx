import { createStore, combineReducers, applyMiddleware, compose } from 'redux';
import { productsReducer } from '../reducers/productReducer';
import { modalDataReducer } from '../reducers/modalDataReducer';
import { isModalOpenReducer } from '../reducers/isModalOpenReducer';
import { favouriteItemsReducer2 } from '../reducers/favouriteItemsReduser2';
import { cartItemsReducer } from '../reducers/cartItemsReducer';
import thunk from 'redux-thunk';

const rootReducer = combineReducers({
    products: productsReducer,
    modalData: modalDataReducer,
    isModalOpen: isModalOpenReducer,
    favouriteItems2: favouriteItemsReducer2,
    cartItems: cartItemsReducer,
});

const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

export const store = createStore(rootReducer, composeEnhancers(applyMiddleware(thunk)));
