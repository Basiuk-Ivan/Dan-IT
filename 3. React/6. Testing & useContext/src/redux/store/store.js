import { configureStore } from '@reduxjs/toolkit';
import productsSlice from '../slice/product';
import modalDataSlice from '../slice/modalData';
import favouriteItemsSlice from '../slice/favouriteItems';
import cartItemsSlice from '../slice/cartItems';
import thunk from 'redux-thunk';

export const store = configureStore({
    reducer: {
        products: productsSlice,
        modalData: modalDataSlice,
        favouriteItems: favouriteItemsSlice,
        cartItems: cartItemsSlice,
    },
    middleware: [thunk],
});
