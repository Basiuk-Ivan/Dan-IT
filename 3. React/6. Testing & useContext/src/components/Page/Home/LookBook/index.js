import LookBook from './LookBook'

export default LookBook