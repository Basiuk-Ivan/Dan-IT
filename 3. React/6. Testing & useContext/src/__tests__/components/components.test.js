import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import { store } from '../../redux/store/store';
import Header from '../../components/Header';
import LookBook from '../../components/Page/Home/LookBook';
import Footer from '../../components/Page/Home/Footer/Footer';

describe('snapshot Components', () => {
    test('snapshot Header', () => {
        const header = render(
            <BrowserRouter>
                <Provider store={store}>
                    <Header />
                </Provider>
            </BrowserRouter>
        );
        expect(header).toMatchSnapshot();
    });
    test('snapshot Footer', () => {
        const footer = render(
            <BrowserRouter>
                <Provider store={store}>
                    <Footer />
                </Provider>
            </BrowserRouter>
        );
        expect(footer).toMatchSnapshot();
    });
    test('snapshot LookBook', () => {
        const lookBook = render(
            <BrowserRouter>
                <Provider store={store}>
                    <LookBook />
                </Provider>
            </BrowserRouter>
        );
        expect(lookBook).toMatchSnapshot();
    });
});
