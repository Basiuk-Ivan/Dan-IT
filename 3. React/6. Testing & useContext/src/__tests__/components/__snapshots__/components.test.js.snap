// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Components snapshot Footer 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="footer"
      >
        <div
          class="footerContact"
        >
          <div
            class="footerLogo"
          >
            <img
              alt="footerLogo"
              src="footerLogo.svg"
            />
          </div>
          <div
            class="footerContacts"
          >
            <p>
              Потрібна допомога з замовленням?
            </p>
            <p>
              <a
                href="tel:+380993242484"
              >
                +380 99 324 24 84
              </a>
            </p>
            <p>
              <a
                href="mailto:info@loylen.com"
              >
                info@loylen.com
              </a>
            </p>
          </div>
          <div
            class="footerDesk"
          >
            <p>
              2021 Інтернет-магазин LOYLEN
            </p>
            <p>
              Всі права захищені
            </p>
            <p>
              Розробка дизайну
            </p>
          </div>
        </div>
        <div
          class="footerInfo"
        >
          <div
            class="footerInfoWrapp"
          >
            <div
              class="footerInfoCatalog"
            >
              <div
                class="footerInfoCatalogTitle"
              >
                Каталог
              </div>
              <ul
                class="footerInfoText"
              >
                <li>
                  <a
                    href="#"
                  >
                    Новинки
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    Розпродаж
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    Готові зразки
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    Всі товари
                  </a>
                </li>
              </ul>
            </div>
            <div
              class="footerInfoCustomer"
            >
              <div
                class="footerInfoCatalogTitle"
              >
                Покупець
              </div>
              <ul
                class="footerInfoText"
              >
                <li>
                  <a
                    href="#"
                  >
                    Особистий кабінет
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    Таблиця розмірів
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    Доставка і оплата
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    Повернення
                  </a>
                </li>
              </ul>
            </div>
            <div
              class="footerInfoAbout"
            >
              <div
                class="footerInfoCatalogTitle"
              >
                Про нас
              </div>
              <ul
                class="footerInfoText"
              >
                <li>
                  <a
                    href="#"
                  >
                    Про бренд
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    Контакти
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    Оферта
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    Політика
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="footer"
    >
      <div
        class="footerContact"
      >
        <div
          class="footerLogo"
        >
          <img
            alt="footerLogo"
            src="footerLogo.svg"
          />
        </div>
        <div
          class="footerContacts"
        >
          <p>
            Потрібна допомога з замовленням?
          </p>
          <p>
            <a
              href="tel:+380993242484"
            >
              +380 99 324 24 84
            </a>
          </p>
          <p>
            <a
              href="mailto:info@loylen.com"
            >
              info@loylen.com
            </a>
          </p>
        </div>
        <div
          class="footerDesk"
        >
          <p>
            2021 Інтернет-магазин LOYLEN
          </p>
          <p>
            Всі права захищені
          </p>
          <p>
            Розробка дизайну
          </p>
        </div>
      </div>
      <div
        class="footerInfo"
      >
        <div
          class="footerInfoWrapp"
        >
          <div
            class="footerInfoCatalog"
          >
            <div
              class="footerInfoCatalogTitle"
            >
              Каталог
            </div>
            <ul
              class="footerInfoText"
            >
              <li>
                <a
                  href="#"
                >
                  Новинки
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Розпродаж
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Готові зразки
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Всі товари
                </a>
              </li>
            </ul>
          </div>
          <div
            class="footerInfoCustomer"
          >
            <div
              class="footerInfoCatalogTitle"
            >
              Покупець
            </div>
            <ul
              class="footerInfoText"
            >
              <li>
                <a
                  href="#"
                >
                  Особистий кабінет
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Таблиця розмірів
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Доставка і оплата
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Повернення
                </a>
              </li>
            </ul>
          </div>
          <div
            class="footerInfoAbout"
          >
            <div
              class="footerInfoCatalogTitle"
            >
              Про нас
            </div>
            <ul
              class="footerInfoText"
            >
              <li>
                <a
                  href="#"
                >
                  Про бренд
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Контакти
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Оферта
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  Політика
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`snapshot Components snapshot Header 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <div
          class="headerMenu"
        >
          <ul
            class="headerMenuList"
          >
            <li
              class="headerMenuItem"
            >
              <a
                class="headerMenuLink"
                href="/"
              >
                Home
              </a>
            </li>
            <li
              class="headerMenuItem"
            >
              <a
                class="headerMenuLink"
                href="#"
              >
                LookBook
              </a>
            </li>
            <li
              class="headerMenuItem"
            >
              <a
                class="headerMenuLink"
                href="#"
              >
                Про бренд
              </a>
            </li>
            <li
              class="headerMenuItem"
            >
              <a
                class="headerMenuLink"
                href="#"
              >
                Інформація
              </a>
            </li>
          </ul>
        </div>
        <div
          class="headerLogo"
        >
          <a
            href="/"
          >
            <img
              alt="logo"
              src="logo.svg"
            />
          </a>
        </div>
        <div
          class="headerInfo"
        >
          <div
            class="headerSearch"
          >
            <img
              alt="search"
              src="search.svg"
            />
          </div>
          <div
            class="headerAccount"
          >
            <img
              alt="account"
              src="account.svg"
            />
          </div>
          <div
            class="headerFavourite"
          >
            <a
              href="/favourite"
            >
              <img
                alt="vafourite"
                src="favourite.svg"
              />
            </a>
            0
          </div>
          <div
            class="cart"
          >
            <a
              href="/cart"
            >
              <img
                alt="cart"
                src="cart.svg"
              />
            </a>
            0
          </div>
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <div
        class="headerMenu"
      >
        <ul
          class="headerMenuList"
        >
          <li
            class="headerMenuItem"
          >
            <a
              class="headerMenuLink"
              href="/"
            >
              Home
            </a>
          </li>
          <li
            class="headerMenuItem"
          >
            <a
              class="headerMenuLink"
              href="#"
            >
              LookBook
            </a>
          </li>
          <li
            class="headerMenuItem"
          >
            <a
              class="headerMenuLink"
              href="#"
            >
              Про бренд
            </a>
          </li>
          <li
            class="headerMenuItem"
          >
            <a
              class="headerMenuLink"
              href="#"
            >
              Інформація
            </a>
          </li>
        </ul>
      </div>
      <div
        class="headerLogo"
      >
        <a
          href="/"
        >
          <img
            alt="logo"
            src="logo.svg"
          />
        </a>
      </div>
      <div
        class="headerInfo"
      >
        <div
          class="headerSearch"
        >
          <img
            alt="search"
            src="search.svg"
          />
        </div>
        <div
          class="headerAccount"
        >
          <img
            alt="account"
            src="account.svg"
          />
        </div>
        <div
          class="headerFavourite"
        >
          <a
            href="/favourite"
          >
            <img
              alt="vafourite"
              src="favourite.svg"
            />
          </a>
          0
        </div>
        <div
          class="cart"
        >
          <a
            href="/cart"
          >
            <img
              alt="cart"
              src="cart.svg"
            />
          </a>
          0
        </div>
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`snapshot Components snapshot LookBook 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="lookBook"
      >
        <div
          class="lookBookImg"
        >
          <img
            alt="lookBook"
            src="lookBook.png"
          />
        </div>
        <div
          class="lookBookInfo"
        >
          <div
            class="lookBookInfoWrapp"
          >
            <div
              class="lookBookTitle"
            >
              LOOKBOOK
            </div>
            <div
              class="lookBookText"
            >
              При створені коллекцій ми враховуємо, щоб всі вироби поєднувались. Це дозволяє підібрати повний образ на всі випадки життя.
            </div>
            <button
              class="lookBookButton"
            >
              Дивитись
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="lookBook"
    >
      <div
        class="lookBookImg"
      >
        <img
          alt="lookBook"
          src="lookBook.png"
        />
      </div>
      <div
        class="lookBookInfo"
      >
        <div
          class="lookBookInfoWrapp"
        >
          <div
            class="lookBookTitle"
          >
            LOOKBOOK
          </div>
          <div
            class="lookBookText"
          >
            При створені коллекцій ми враховуємо, щоб всі вироби поєднувались. Це дозволяє підібрати повний образ на всі випадки життя.
          </div>
          <button
            class="lookBookButton"
          >
            Дивитись
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
